{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings.py

```diff
@@ -44,8 +44,8 @@
 if DEBUG:
     INSTALLED_APPS += ('debug_toolbar',)
 
-{% if cookiecutter.use_celery == "y" -%}
-INSTALLED_APPS += ('djcelery',)
+{% if cookiecutter.use_rq == "y" -%}
+INSTALLED_APPS += ('django_rq',)
```

Comment:
does INSTALLED_APPS not exist if you don't use_rq? it would be nice to have even if it's empty